packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -27,7 +26,5 @@ describe('plugin-devices', () => {
     describe('#resetLogoutTimer()', () => {});
     describe('#unregister()', () => {});
     describe('#waitForRegistration()', () => {});
-
-    it('temporary pass flag', () => assert.isTrue(true));
```

Comment:
Surprised the test library didn't fail due to no tests, but seems to have passed 👍